.drone.yml

```diff
@@ -0,0 +1,48 @@
+workspace:
+  base: /srv/app
+  path: src/github.com/go-gitea/gitea
+
+pipeline:
+  testing:
+    image: webhippie/golang:edge
+    pull: true
+    environment:
+      CGO_ENABLED: 0
+    commands:
+      - make clean
```

Comment:
Just add a line like `apk -U add openssh-client` to fix the blocking about missing `ssh-keygen`.